css-brochure-form/styles.css

```diff
@@ -0,0 +1,67 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+h2 {
+  font-family: "Brandon Grotesque", sans-serif;
+  color: #e34f3d;
+  font-weight: 500;
+  font-size: 15pt;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+}
+
+label {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-weight: 500;
+  font-size: 10pt;
```

Comment:
Moving forward, let's avoid using `pt` for any font sizing. We do want to stick with either `px` or `em` or `rem`